signer-cloud-server/src/main/java/com/wultra/signercloud/server/document/DocumentStatus.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Signer Cloud
+ * Copyright (C) 2025 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.signercloud.server.document;
+
+/**
+ * Enum representing the status of a {@link Document} in the signing process.
+ *
+ * @author Michal Rozehnal, michal.rozehnal@wultra.com
+ */
+public enum DocumentStatus {
+    /**
+     * Document is uploaded and is waiting for signature
```

Comment:
Probably add a point at the end of the sentence.
```suggestion
     * Document is uploaded and is waiting for signature.
```

Reply:
Yes, updated.